gtfe/__init__.py

```diff
@@ -8,6 +9,9 @@
 
 RFC2822_FORMAT = '%a, %d %b %Y %T %z'
 
+def FALLAO_():
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename function "FALLAO_" to match the regular expression ^[a-z_][a-z0-9_]{2,30}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=python%3AS1542)